.github/workflows/test-cirros-bionic.yaml

```diff
@@ -0,0 +1,199 @@
+name: Test CirrOS build
```

Comment:
Drop whole file please - it can be part of previous one.

Reply:
Sounds good. I initially added this file as a separate workflow because I was not sure if we wanted to preserve the built images: save on artifact storage space. Artifact storage space does not look like a concern for public repositories however so I can drop this file.